Dockerfile.modules

```diff
@@ -0,0 +1,14 @@
+FROM debian:jessie
```

Comment:
I'd prefer using the same base image as freeswicth for dependencies like libc,  abi compatibility etc.